pages/sdk/protocol-kit/guides/safe-deployment/address-replication.mdx

```diff
@@ -0,0 +1,173 @@
+import { Steps } from "nextra/components";
+
+# Safe Address Replication Across Chains
+
+In this guide, you will learn how to replicate a Safe address across different chains using the Protocol Kit. This process includes initializing the Protocol Kit, configuring the Safe, predicting the Safe address on different chains, and executing the deployment.
+
+We will demonstrate how to deploy a Safe account on both the Sepolia and Chiado chains with the same address using the Protocol Kit.
+
+For more detailed information, see the [Protocol Kit Reference](../../../protocol-kit/reference/safe.mdx).
+
+## Prerequisites
+
+- [Node.js and npm](https://docs.npmjs.com/downloading-and-installing-node-js-and-npm)
+
+## Install dependencies
+
+First, you need to install the Protocol Kit.
+
+```bash
+pnpm add @safe-global/protocol-kit
```

Comment:
pnpm instead yarn or npm?

Reply:
I've updated the installation snippet to align with our new documentation format:

```
<CH.Section>
  <CH.Code style={{ boxShadow: 'none' }}>
    ```bash pnpm
    pnpm add @safe-global/protocol-kit
    ```

    ```bash npm
    npm install @safe-global/protocol-kit
    ```

    ```bash yarn
    yarn add @safe-global/protocol-kit
    ```
  </CH.Code>
</CH.Section>
```